avocado/core/spawners/common.py

```diff
@@ -22,7 +22,7 @@ class BaseSpawner:
 
     @staticmethod
     def is_task_alive(task):
-        pass
+        raise SpawnerException("You need to implement this method.")
```

Comment:
Isn't `NotImplementedError` a better option here? One other possibility is using `abc`s.

Reply:
@clebergnu yes, I agree. It will be fixed on v2.

Will more comments come? Or can I send a v2?

Reply:
Oh sorry, I'm still reviewing it. Will have it over soon with a formal GH "review" posted.